src/main/java/io/confluent/examples/streams/PageViewRegionLambdaExample.java

```diff
@@ -27,15 +27,11 @@
 import org.apache.kafka.streams.KeyValue;
 import org.apache.kafka.streams.StreamsBuilder;
 import org.apache.kafka.streams.StreamsConfig;
-import org.apache.kafka.streams.kstream.KStream;
-import org.apache.kafka.streams.kstream.KTable;
-import org.apache.kafka.streams.kstream.Produced;
-import org.apache.kafka.streams.kstream.TimeWindows;
-import org.apache.kafka.streams.kstream.Windowed;
+import org.apache.kafka.streams.kstream.*;
```

Comment:
Nit: want to avoid wildcard imports?

Reply:
Yep. I didn't notice that happened. I'll fix them.